spring-cloud-gcp-data-firestore/src/test/java/org/springframework/cloud/gcp/data/firestore/it/FirestoreRepositoryIntegrationTests.java

```diff
@@ -44,11 +47,16 @@
 	UserRepository userRepository;
 	//end::autowire[]
 
+	//tag::autowire_tx_manager[]
+	@Autowired
+	ReactiveFirestoreTransactionManager txManager;
+	//end::autowire_tx_manager[]
+
 	@BeforeClass
 	public static void checkToRun() throws IOException {
 		assumeThat("Firestore-sample tests are disabled. "
 				+ "Please use '-Dit.firestore=true' to enable them. ",
-				System.getProperty("it.firestore"), is("true"));
+				"true", is("true"));
```

Comment:
Testing-only change? Code needs to go back to checking `System.getProperty("it.firestore")`.

Reply:
done